diff --git a/tensorflow/contrib/cmake/tf_core_cpu.cmake b/tensorflow/contrib/cmake/tf_core_cpu.cmake
index a54cbff..d99c0da 100644
--- a/tensorflow/contrib/cmake/tf_core_cpu.cmake
+++ b/tensorflow/contrib/cmake/tf_core_cpu.cmake
@@ -20,8 +20,8 @@ file(GLOB_RECURSE tf_core_cpu_srcs
     "${tensorflow_source_dir}/tensorflow/cc/saved_model/*.cc"
     "${tensorflow_source_dir}/tensorflow/core/common_runtime/*.h"
     "${tensorflow_source_dir}/tensorflow/core/common_runtime/*.cc"
-    "${tensorflow_source_dir}/tensorflow/core/debug/*.h"
-    "${tensorflow_source_dir}/tensorflow/core/debug/*.cc"
+#    "${tensorflow_source_dir}/tensorflow/core/debug/*.h"
+#    "${tensorflow_source_dir}/tensorflow/core/debug/*.cc"
     "${tensorflow_source_dir}/tensorflow/core/distributed_runtime/server_lib.h"
     "${tensorflow_source_dir}/tensorflow/core/distributed_runtime/server_lib.cc"
     "${tensorflow_source_dir}/tensorflow/core/graph/*.h"
